server/models/Notification.ts

```diff
@@ -182,6 +184,47 @@ export class Notification extends Model<InferAttributes<Notification>, InferCrea
     return notifications.map(us => us.UserId);
   }
 
+  static async getUnsubscribers(_where: {
+    type?: ActivityClasses | ActivityTypes;
+    CollectiveId?: number;
+    UserId?: number | number[];
+  }) {
+    debug('getUnsubscribers', { _where });
+    const getUsers = notifications => notifications.map(notification => notification.User);
+
+    const include = [{ model: models.User }];
+    const where = { active: false };
+    if (_where.UserId) {
+      where['UserId'] = _where.UserId;
+    }
+
+    const collective = _where.CollectiveId && (await models.Collective.findByPk(_where.CollectiveId));
+    if (collective) {
+      where['CollectiveId'] = compact([collective.id, collective.ParentCollectiveId, collective.HostCollectiveId]);
+      // If user is informed, also fetch global settings
```

Comment:
I don't really understand what `informed` means in this context? Overall this entire function is pretty difficult for me to understand, it would really benefit from adding some comments to explain what we're doing at each stage.

Reply:
Sorry about the language. The idea is that if UserId is passed as an argument, we should also consider global notification settings: basically, the setting of an UserId unsubscribe to a whole Activity Type without any CollectiveId passed to it.